src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -81,7 +81,6 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
   // Between the changes and the default values so we don't submit an unchanged field
   const mutationParams = diff(defaultEditSpawnHostState, editSpawnHostState);
   console.log({ mutationParams });
```

Comment:
🙅

Reply:
Thanks this will be removed when i wire up the modal I left it to avoid an error about the unused variable.